src/variables.jl

```diff
@@ -389,3 +408,35 @@ resultvalue(v::VariableRef) = MOI.get(v.m, MOI.VariablePrimal(), v)
 hasresultvalues(m::Model) = MOI.canget(m, MOI.VariablePrimal(), VariableRef)
 
 @Base.deprecate setvalue(v::VariableRef, val::Number) setstart(v, val)
+
+"""
+    addvariable(m::Model, v::AbstractVariable, name::String="")
```

Comment:
The signature in the documentation should match the method you're defining. (`ScalarVariable` vs. `AbstractVariable`)